src/main/java/org/neo4j/sql2cypher/SQLToCypher.java

```diff
@@ -44,38 +44,87 @@
 import org.neo4j.cypherdsl.core.renderer.Configuration;
 import org.neo4j.cypherdsl.core.renderer.Renderer;
 
-import static org.jooq.impl.DSL.createTable;
-
 /**
  * Quick proof of concept of a jOOQ/Cypher-DSL based SQL to Cypher translator
  * @author Lukas Eder
  * @author Michael J. Simons
  */
 public class SQLToCypher {
 
+
     public static void main(String[] args) {
-        System.out.println(new SQLToCypher().convert("""
+        System.out.println(new SQLToCypher().using(new Config()).convert("""
             SELECT t.a, t.b
             FROM my_table AS t
             WHERE t.a = 1
             """));
     }
 
-    // Meta data can be added via .meta(createTable("t").column("a", SQLDataType.INTEGER))
-    private final Parser parser = DSL.using(SQLDialect.DEFAULT).parser();
-    private final Renderer renderer = Renderer.getRenderer(Configuration.prettyPrinting());
-    private final Map<Table<?>, Node> tables = new HashMap<>();
+    private Config config = new Config();
+
+    static class Context {
+        private final Map<Table<?>, Node> tables = new HashMap<>();
+
+        void addTable(Table<?> table, Node node) {
+            this.tables.put(table,node);
+        }
+
+        private Node getNode(Table<?> t) {
+            return tables.get(t);
+        }
+    }
```

Comment:
Is extracting this `Context` truly better? The jOOQ parser had this before. And then *all* the 581734 parser methods had to pass around `Context`, when in fact, the context could have been just `this`.